recipe/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "anaconda-packaging-utils" %}
+{% set version = "v0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/anaconda-distribution/anaconda-packaging-utils/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 6a440a85ca29452c2900f272c971b505c56569a955be188d660dc31d9fdf044c
+
+build:
+  number: 0
+  noarch: python
+  script: pip install . --no-deps --no-build-isolation -vv
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - wheel
+  run:
+    - python >=3.11
+    - conda
+    - jsonschema
+    - types-jsonschema
+    - percy >=0.1.1
+    - PyGithub
+    - pyyaml
+    - types-pyyaml
+    - requests
+    - types-requests
+
+test:
+  imports:
+    - anaconda_packaging_utils
+  source_files:
+    - anaconda-packaging-utils-config-template.yaml  # Used for self-test
+    - anaconda_packaging_utils/tests/
+  requires:
+    - pip
+    - pytest
+    - pytest-xdist
+  commands:
+    - pip check
+    - python -m pytest -n auto --capture=no anaconda_packaging_utils/tests/ -k "not test_subshell"  # [win]
```

Comment:
If `test_subshell` doesn't work on Windows, they should be skipped straight in the source instead of here.

Reply:
I'll make that change, but that does mean I have to cut a new version of the project and update the recipe, which is annoying IMO.

Realistically I don't know if it is _that_ important that we have windows support for an internal tooling project. For that matter, we probably also don't need s390, PPC, or really anything other than linux64 and MacOS

Reply:
On second thought, I suppose we might as well keep the other architectures in the event that we do end up having to develop tools that run on those platforms.